macros/models/_sector/dex/trader_joe_compatible_trades.sql

```diff
@@ -0,0 +1,113 @@
+{% macro trader_joe_compatible_v2_trades(
+    blockchain = null
+    , project = null
+    , version = null
+    , Pair_evt_Swap = null
+    , Factory_evt_PoolCreated = null
+    , pair_column_name = 'LBPair'
+    )
+%}
+WITH dexs AS
+(
+    SELECT
+        t.evt_block_number AS block_number
+        , t.evt_block_time AS block_time
+        , t.to AS taker
```

Comment:
```suggestion
        , t.recipient AS taker
```

Reply:
Ah right, totally missed that 😅
Thank you for the help 🙏